algolia/suggestions/client_config_test.go

```diff
@@ -0,0 +1,37 @@
+package suggestions
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig_Unmarshal(t *testing.T) {
+	var c IndexConfiguration
+	data := []byte(`{"languages":["ja"]}`)
+	err := json.Unmarshal(data, &c)
+	require.NoError(t, err)
+	require.False(t, c.Languages.IsBool)
+	require.Equal(t, []string{"ja"}, c.Languages.StringArray)
+
+	c = IndexConfiguration{}
+	data = []byte(`{"languages":false}`)
+	err = json.Unmarshal(data, &c)
+	require.NoError(t, err)
+	require.False(t, c.Languages.Bool)
+
+	c = IndexConfiguration{}
+	data = []byte(`{}`)
+	err = json.Unmarshal(data, &c)
+	require.NoError(t, err)
+	require.False(t, c.Languages.IsBool)
+}
+
+func TestConfig_Marshal(t *testing.T) {
+	config := IndexConfiguration{
+		IndexName: "my_index",
+	}
+	json, _ := json.Marshal(config)
+	t.Log(string(json))
```

Comment:
what does this test ?

Reply:
Nice catch, I used it for debugging purposes. Removed.